app/_components/ai-multi-models.tsx

```
@@ -0,0 +1,118 @@
+    }
+
+
+    return (
+        <div
+            className='flex flex-1 h-[77dvh] border-b border-gray-200 dark:border-gray-700'
+        >
+            {
+                aiModelList.map((model, index) => (
+                    <div
+                        className={`flex flex-col h-full border-r border-gray-200 dark:border-gray-700 overflow-auto ${model.enable ? 'flex-1 min-w-[400px]' : 'flex-none w-[100px]'}`}
+                    >
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)